starter_code/api/spotify.js

```diff
@@ -0,0 +1,17 @@
+const SpotifyWebApi   = require('/../spotify-web-api-node');
+
+const spotify = new SpotifyWebApi({
+    clientId : 'd9173cf7be274a2280d8171adf8aa2d7',
+    clientSecret : '39cc2a98052c4b4781bfe2b06528f7be',
+  });
+
+spotify.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.